Drafts/1NNN-VLP.md

```diff
@@ -0,0 +1,86 @@
+# New proposed syntax for mixins
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | Victor Porton <porton@narod.ru>                                              |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | Draft                                                           |
+
+## Abstract
+
+The programmer sometimes needs to instantiate mixin templates with the same arguments repeatedly, like:
+
+```
+mixin template A(int x, float y) {
+  // ...
+}
+
+mixin template B(int x, float y) {
+  // ...
+}
+
+I propose to shorten the above code with the following syntax:
+
+template composite(int x, float y) {
+  mixin A {
+    // ...
+  }
+  mixin B {
+    // ...
+  }
+}
+
+// ...
+
+alias ourMixins = composite(1, 2.0)
+
+struct Test {
+  // put two mixins into Test
+  ourMixins.A;
+  ourMixins.B;
```

Comment:
This use-site syntax does not show the reader that arbitrary code is being injected. For consistency it should be `mixin ourMixins.B;`.
With that change, this can be done already by changing e.g. `mixin A` in the template to `mixin template A()`.